src/main/groovy/com/ullink/ProjectFileParser.groovy

```diff
@@ -40,25 +40,42 @@ class ProjectFileParser {
         }
     }
 
-    def renameExtension(def file, def newExtension) {
-        FilenameUtils.removeExtension(file) + newExtension
+    File getDotnetAssemblyFile() {
+        if (properties.TargetPath) {
+            project.file(properties.TargetPath)
+        }
     }
 
-    File getDotnetAssemblyFile() {
-        project.file(properties.TargetPath)
+    List getDotnetDebugFile() {
+        if (properties.DebugSymbols) {
+            if (eval._DebugSymbolsOutputPath) {
+                eval._DebugSymbolsOutputPath.collect { new File (it.FullPath) }

```

Comment:
that's why the return type is changed to List, or you think just getting the first one is better?